src/main/java/anna_gontari/CopyingFile.java

```diff
@@ -0,0 +1,37 @@
+package anna_gontari;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+//Write a program that copies the contents of one text file to another file. Have the user enter the names of the  target file.
+public class CopyingFile {
+    public static void main(String[] args) {
+        copyFile();
+    }
+    public static void copyFile() {
+        System.out.println("Please, enter file's path from which you want to copy information: ");
+        Scanner scanner = new Scanner(System.in);
+        String input1 = scanner.nextLine();
+        System.out.println("Please enter new file name:");
+        String input2 = scanner.nextLine();
+
+        File path = new File("New Folder/copy Files/new Files");
+        try {
+            if (path.mkdir() == true) {
```

Comment:
path.mkdir() can be only true or false 

path.mkdir() returns true or false , so the third else is redundant, there is not 3-th case